libs/common/src/sql-db-entities/fee-record/fee-record.entity.ts

```diff
@@ -237,6 +238,34 @@ export class FeeRecordEntity extends AuditableBaseEntity {
     }
   }
 
+  /**
+   * Updates the fee record with a correction
+   * @param param - The update parameters
+   * @param param.correctedValues - The corrected values, each field can be null if not being corrected
+   * @param param.requestSource - The request source
+   */
+  public updateWithCorrection({ requestSource, correctedValues }: UpdateWithCorrectionParams): void {
+    this.status = FEE_RECORD_STATUS.TO_DO_AMENDED;
+
+    if (correctedValues.facilityUtilisation !== null) {
```

Comment:
would a switch here without breaks be more efficient?

Reply:
I'm not really sure what you mean, each if here is operating against a different field and they can happen in any combination so I can't see how one would write this as a switch.